ocw_data_parser/utils_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import pytest
+from ocw_data_parser.ocw_data_parser_test import ocw_parser
+from ocw_data_parser.utils import update_file_location, get_binary_data, is_json, get_correct_path, load_json_file, print_error, print_success, safe_get, find_all_values_for_key
+
+
+def test_update_local_file_location(ocw_parser):
+    """
+    Extract local course media, update the location of one of the files 
+    and then assert that the location has indeed changed
+    """
+    ocw_parser.extract_media_locally()
+    if len(ocw_parser.master_json["course_files"]) > 0:
+        test_file = ocw_parser.master_json["course_files"][0]
+        original_location = test_file["file_location"]
+        update_file_location(ocw_parser.master_json,
+                             "test_location", obj_uid=test_file["uid"])
+        assert original_location != ocw_parser.master_json["course_files"][0]["file_location"]
+    else:
+        fail("test course has no local media to test")
+
+
+def test_update_foreign_file_location(ocw_parser):
+    """
+    Extract foreign course media, update the location of one of the files 
+    and then assert that the location has indeed changed
+    """
+    ocw_parser.extract_foreign_media_locally()
+    if len(ocw_parser.master_json["course_foreign_files"]) > 0:
+        test_file = ocw_parser.master_json["course_foreign_files"][0]
+        original_location = test_file["file_location"]
+        original_filename = test_file["link"].split("/")[-1]
+        update_file_location(ocw_parser.master_json,
+                             os.path.join("test_location/", original_filename))
+        assert original_location != ocw_parser.master_json["course_foreign_files"][0]["file_location"]
+    else:
+        fail("test course has no foreign media to test")
+
+
+def test_get_binary_data_none(ocw_parser):
+    """
+    Find the first file without a datafield property and attempt to get the binary data from it
+    """
+    if len(ocw_parser.master_json["course_files"]) > 0:
+        found = False
+        for media in ocw_parser.master_json["course_files"]:
+            if "_datafield_image" not in media and "_datafield_file" not in media:
+                found = True
+                data = get_binary_data(media)
+                assert data is None
+        if not found:
```

Comment:
We can rewrite as `assert found, "test course has no file without a datafield property"`.

The `if` statement above can also be rewritten as:

    assert len(ocw_parser.master_json["course_files"]) > 0
    found = False
    for media in ocw_parser.master_json["course_files"]:
        ...